### PR TITLE
Delete dead code.

### PR DESCRIPTION
**Pull request by KirillOsenkov**
@Forgind I keep my promises ;)

### CODE REVIEW

File: src/Build/Logging/BinaryLogger/BuildEventArgsWriter.cs [hunk 1]
```diff
@@ -705,12 +705,6 @@ private static BuildEventArgsFieldFlags GetBuildEventArgsFieldFlags(BuildEventAr
                 flags |= BuildEventArgsFieldFlags.SenderName;
             }
 
-            // ThreadId never seems to be used or useful for anything.
-            //if (e.ThreadId > 0)
```
Comment (Forgind) [inline src/Build/Logging/BinaryLogger/BuildEventArgsWriter.cs:709]: 💗

File: src/Build/Logging/BinaryLogger/BuildEventArgsReader.cs [hunk 2]
```diff
@@ -51,8 +51,6 @@ public class BuildEventArgsReader : IDisposable
             typeof(BuildEventArgs).GetField("threadId", BindingFlags.Instance | BindingFlags.NonPublic);
         private static FieldInfo buildEventArgsFieldSenderName =
             typeof(BuildEventArgs).GetField("senderName", BindingFlags.Instance | BindingFlags.NonPublic);
-        private static FieldInfo buildEventArgsFieldTimestamp =
```
Comment (Forgind) [inline src/Build/Logging/BinaryLogger/BuildEventArgsReader.cs:54]: This doesn't have to do with the timestamp? That still looks like something that can be written.

**Review by Forgind — APPROVED** [review]
Thanks for remembering!

File: src/Build/Logging/BinaryLogger/BuildEventArgsReader.cs (hunk 2 above)
Comment (KirillOsenkov) [reply src/Build/Logging/BinaryLogger/BuildEventArgsReader.cs:54]: yes this is separate, was never used, I decided to pile it on

**Review by rainersigwald — APPROVED** [review]

**Pull request merged**